packages/web-app-webfinger/src/views/Resolve.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div
+    class="webfinger-resolve oc-height-viewport oc-flex oc-flex-column oc-flex-center oc-flex-middle"
+  >
+    <div class="oc-card oc-card-body oc-text-center oc-width-large">
+      <template v-if="hasError">
+        <h2 key="webfinger-resolve-error">
```

Comment:
Don't we need a `h1` to be semantically correct?

Reply:
added the `h1` as screenreader-only page title, like we do in basically all other views.